sklearn/model_selection/tests/test_search.py

```diff
@@ -336,6 +507,21 @@ def test_grid_search_one_grid_point():
     assert_array_equal(clf.dual_coef_, cv.best_estimator_.dual_coef_)
 
 
+@nottest
```

Comment:
Because of the TODO below. I'd forgotten about that, but have been given an answer on how to deal with it.